docs/components/camera/transform.md

```diff
@@ -74,59 +74,66 @@ The following attributes are available for `transform` views:
 The following are the transformation objects available for the `pipeline`:
 
 {{< tabs >}}
-{{% tab name="Identity"%}}
+{{% tab name="Classifications" %}}
 
-The Identity transform does nothing to the image.
-You can use this transform to change the underlying camera source's intrinsic parameters or stream type, for example.
+Classifications overlay text from the `GetClassifications` method of the [Vision Service](../../../services/vision) onto the image.
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
-    "type": "identity"
+    "type": "classifications",
+    "attributes": {
+        "classifier_name": string,
+        "confidence_threshold": float
+    }
 }
 ```
 
 **Attributes:**
 
-- None
+- `classifier_name`: The name of the classifier in the [Vision Service](../../../services/vision).
+- `confidence_threshold`: The threshold above which to display classifications.
 
 {{% /tab %}}
-{{% tab name="Rotate" %}}
 
-The Rotate transformation rotates the image by 180 degrees.
-This feature is useful for when the camera is installed upside down on your robot.
+{{% tab name="Depth Edges" %}}
+
+The Depth Edges transform creates a canny edge detector to detect edges on an input depth map.
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
-    "type": "rotate",
-    "attributes": { }
+    "type": "depth_edges",
+    "attributes": {
+        "high_threshold_pct": float,
+        "low_threshold_pct": float,
+        "blur_radius_px": float
+    }
 }
 ```
 
 **Attributes:**
 
-- None
+- `high_threshold_pct`: The high threshold of ??? : between 0.0 - 1.0.
```

Comment:
??? I think needs to be clarified here and below! Sorry as I don't know what it is either

Reply:
Oh amazing 😂

Reply:
basing on this https://en.wikipedia.org/wiki/Canny_edge_detector